src/syzygy/syzygy.cc

```diff
@@ -0,0 +1,1676 @@
+/*
+ * Original from cfish tbprobe.c
+ * Copyright (c) 2013-2018 Ronald de Man
+ * This file may be redistributed and/or modified without restrictions.
```

Comment:
What does that mean? It is not GPL? It's not allowed by GPL to have non-GPL files.
And if it's GPL, we need banner.

Reply:
That's up to @Tilps

Reply:
This file was originally taken from a project licensed overall by GPL, but this file is licensed instead in a more 'informal' way.  GPL compatible list, has informal licensing as one of its compatible with entries, and given it previously existed embedded in an GPL codebase, that provides evidence that the authors intention was for the informal licensing to be GPL compatible.

We could put a GPL banner on here stating that the modifications are licensed under GPL with exception, like the majority of our codebase. But we should retain this section.